web/src/features/hfiCalculator/components/IntensityGroupCell.tsx

```diff
@@ -1,6 +1,5 @@
 import { makeStyles, TableCell } from '@material-ui/core'
 import { fireTableStyles } from 'app/theme'
-import { intensityGroupColours } from 'features/hfiCalculator/components/meanIntensity'
```

Comment:
Why the changes to IntensityGroupCell? Shouldn't it just be zone code?

Reply:
This was something I missed on the previous ticket I worked on that Tess just asked to quickly fix on this one.

Reply:
Ok I'm cool with it if @tessmgood is